scripts/create-kubestellar-demo-env.sh

```diff
@@ -146,26 +208,42 @@ images=("ghcr.io/loft-sh/vcluster:0.16.4"
 
 for image in "${images[@]}"; do
     (
-        docker pull "$image" && kind load docker-image "$image" --name kubeflex
+        if [ "$k8s_platform" == "kind" ]; then
+            docker pull "$image" && kind load docker-image "$image" --name kubeflex
+        else
+            docker pull "$image" && k3d image import "$image" --cluster kubeflex
+        fi
     ) &
 done
+wait
 
-echo -e "\nStarting the process to install KubeStellar core: kind-kubeflex..."
-helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart \
-    --version $kubestellar_version \
-    --set-json='ITSes=[{"name":"its1"}]' \
-    --set-json='WDSes=[{"name":"wds1"}]' \
-    --set-json='verbosity.default=5'
+echo -e "\nStarting the process to install KubeStellar core: $k8s_platform-kubeflex..."
+if [ "$k8s_platform" == "kind" ]; then
+    helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart \
+        --version $kubestellar_version \
+        --set-json='ITSes=[{"name":"its1"}]' \
+        --set-json='WDSes=[{"name":"wds1"}]' \
+        --set-json='verbosity.default=5'
+else
+    helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart \
+        --version $kubestellar_version \
+        --set-json='ITSes=[{"name":"its1"}]' \
+        --set-json='WDSes=[{"name":"wds1"},{"name":"wds2", "type":"host"}]' \
```

Comment:
Why make two WDSes in this case and not in the `kind` case? I think that it would be best if this script behaved analogously in the two cases.

Reply:
done

Reply:
the second WDS is needed for the Kueue integration. We can either go back to making 1 for kind and 2 for k3d or keep 2 wds for each. I see no harm in leaving 2 in each version

Reply:
Is this script becoming the install script for all purposes?

Reply:
I think we want the "getting started" configuration to not be more complicated than necessary. If the Kueue integration requires a more complicated setup then I think that should be documented but in a way that does not make the "getting started" configuration more complicated.

Reply:
the intent of this script is to be a baseline for the getting started and the integrations. We should not need an entire set of scripts to run each time we want to run our own getting started or anything in galaxy repo. This script creates the basic "platform" that is KubeStellar. You can experiment with more than just the getting started with this new baseline.

Reply:
If we expand the purpose of this script beyond establishing the configuration used in "get started" then that breaks the equivalence between this script and the step-by-step instructions in "get started". That means that this script needs distinct documentation of its own and the "get startede" page has to explain the difference between what this script does and what the step-by-step instructions do.

The expansion suggested here is still only for kicking tires, not production usage. So I would not say that the configuration produced by this script is 'the basic "platform" that is KubeStellar'.

Reply:
this doesn't break the original getting started - there is an extra WDS - that is all

Reply:
I understand that. My point is that adding extra functionality to this script causes structural problems. The functionality that we expect users to use should be documented. Currently the only documentation of this script is the get-started page, which only documents one WDS. Also, that page asserts equivalence between the script and the step-by-step instructions. See https://github.com/kubestellar/kubestellar/blob/v0.26.0-alpha.1/docs/content/direct/get-started.md?plain=1#L5 ("two ways to create one particular simple configuration) and https://github.com/kubestellar/kubestellar/blob/v0.26.0-alpha.1/docs/content/direct/get-started.md?plain=1#L35 ("the script does the same things as described in the [Step by Step Setup]").

Reply:
I am not necessarily opposed to having just one script that does a bit more, but:

(A) it needs to be documented, and
(B) it needs to stay limited to producing only a demo configuration, or else this becomes a bigger project (indeed one that we. should do, but not as part of this PR).

Reply:
this will be the demo platform from which demos can be constructed upon. It need not go any further than how it is structured now. The steps after this script are to be handled but the specific integrations in places like galaxy. This is intended to be the baseline. And, yes, it can be documented once it is approved

Reply:
We have made exceptions before and I am willing to do so again, but I think that our normal procedure should be to update the documentation in the same PR as the code. It helps reviewers, and it avoids confusing readers of `main`.